### PR TITLE
[Fix](Outfile) upgrade apache-arrow version to 13.0.0

### PR DESCRIPTION
**Pull request by BePPPower**
## Proposed changes

Issue Number: close #xxx

When an invalid endpoint is specified for S3, exporting large data volumes using `select outfile` will cause a BE core dump.
This is due to this bug: https://github.com/apache/arrow/pull/35520 

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by BePPPower** [comment]
run buildall

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Comment by BePPPower** [comment]
run buildall

File: thirdparty/vars.sh [hunk 1]
```diff
@@ -399,10 +412,10 @@ BENCHMARK_MD5SUM="8ddf8571d3f6198d37852bcbd964f817"
 
 # xsimd
 # for arrow-7.0.0, if arrow upgrade, this version may also need to be changed
```
[the remark this answers is not in the document]
Comment (BePPPower) [reply thirdparty/vars.sh]: done

**Review by morningman — APPROVED** [review]
LGTM

**Pull request closed without merging**